docs/faq.rst

```diff
@@ -161,3 +161,40 @@ I am seeing an Exception error, what's wrong?
 
 Have a look at the :ref:`Troubleshooting section <troubleshooting>` to
 see what to do.
+
+If two users modify the same collection offline, how does that conflict get resolved?
+-------------------------------------------------------------------------------------
+
+It depends on which client you use. If you use kinto.js, for instance, you can set up
+three conflict resolution strategies:
+
+* SERVER_WINS: local changes are overridden by remote ones ;
+* CLIENT_WINS: remote changes are overriden by local one ;
+* MANUAL (default): handle them on your own.
+
+There is, of course, a convenient API to handle conflict one by one
+http://kintojs.readthedocs.io/en/latest/api/#resolving-conflicts-manually
+
+Would you recommend redis or postgres?
+--------------------------------------
+
+We recommend using postgres which is most focused lately, but you can still use redis.
```

Comment:
We can rephrase like:

You can use both of them: 

- Redis will let you start easily and you will have a database running in memory which means your database should be smaller than your server RAM. It is a good solution for experimentation and you will also be able to use a Redis cluster to scale in production.

- PostgreSQL is a good solution for a Kinto server and will let you use all the power of PostgreSQL and its tooling.

Do not hesitate to mix both if you can, for instance you can use PostgreSQL for the storage backend and Redis for the permission and cache backends.